### ISSUE TITLE
Add API Documentation via Swagger

### ISSUE THREAD

**Issue by AngelGarcia13** [shakenbytes/riot#26]
See https://dev.to/acanimal/express-api-with-autogenerated-openapi-doc-through-swagger-7na

**Comment by frankely** [comment]
Good call, I really like using open API standards so clients of the API can navigate through it easily.
